### ISSUE TITLE
Visualizer with LabelLUT prints wrong labels when one of the classes is not present in the data.

### ISSUE THREAD

**Issue by daanbl** [isl-org/Open3D-ML#243]
I've encountered a peculiar bug: when visualising a pointcloud using `vis.visualize(data)` with a pre-defined LabelLUT, the visualisation goed wrong when one of the classes is not present in the data.

Example: I ran **RandLA-Net** with pre-trained weights on the **Paris-Lille-3D validation set** (using the code snippet from the readme). It outputs predictions for the classes [1, ..., 9], where 0 is ignored ('unknown class'). Therefore, the data has no points labelled as 0. When I initialise the visualizer with the full set of labels, including class '0', all labels <=4 are shifted 1 down, class 4 is 'missing', and all other classes are shown correctly. When I initialise the visualiser without class 0, it shows everything correctly.

When I set all labels for class '1' to '0' (so now the data does not contain any point of class '1', the same happens: classes 2, 3, and 4 are shifted down, and 4 disappears.

Code:
```
result = pipeline.run_inference(data)

dataset = ml3d.datasets.ParisLille3D(data_dir)
test_split = dataset.get_split("val")
data = test_split.get_data(0)

paris_labels = ml3d.datasets.ParisLille3D.get_label_to_names()
# paris_labels.pop(0)
v = ml3d.vis.Visualizer()
lut = ml3d.vis.LabelLUT()
for val in sorted(paris_labels.keys()):
    lut.add_label(paris_labels[val], val)
v.set_lut("labels", lut)

data_viz = {}
data_viz['name'] = 'randla_npm3d_val'
data_viz['points'] = data['point']
data_viz['labels'] = result['predict_labels'] + 1

v.visualize([data_viz])
```

Example: here the visualisation is correct for class 4.
<img width="1136" alt="Example of correct behaviour" src="https://user-images.githubusercontent.com/12812413/112019644-ea909600-8b2f-11eb-8051-1d790ea9153a.png">

The same data, but now the LabelLUT includes class 0. The data for class 4 is now shown as class 3. Nothing is shown for class 4. Classes >4 are shown correctly (excluded from screenshot for clarity).
<img width="1136" alt="Example of incorrect behaviour" src="https://user-images.githubusercontent.com/12812413/112019707-fd0acf80-8b2f-11eb-9a48-f17bc8f44b01.png">

A quick fix is to set any single label to the missing class (e.g.) 0:
```
data_viz['labels'][0] = 0
```

**Comment by lc-guy** [comment]
I'm getting the same bug. It's difficult to deal with ignored and missing classes like this.
